src/websocket_client.erl

```
@@ -216,13 +216,7 @@ cancel_keepalive_timer(WSReq) ->
                       Reason :: tuple()) -> ok.
 websocket_close(WSReq, HandlerState, Reason) ->
     Handler = websocket_req:handler(WSReq),
-    try Handler:websocket_terminate(Reason, WSReq, HandlerState) of
-        _ ->
-            case Reason of
-                normal -> ok;
-                _      -> error_info(Handler, Reason, HandlerState)
```

Is there a reason you wanted to suppress the error log?